### DISCUSSION TITLE
what does these numbers mean?

### DISCUSSION THREAD

**Discussion by linwryn** [pybamm-team/PyBaMM#2123] [Q&A] [answered]
Hi,
I'm curious that while I'm plotting negative electrode potential it starts with x=0 and ends with x>60. Electrolyte potential starts with x=0 and ends with x>120. Finally, positive electrode potential starts with x<120 and ends with x>150. These are for Chen2020. So, I understand that at the left there is negative electrode until x is bigger than 60 and positive electrode starts at near 120um and in between there is electrolyte. Did I understand right or is there any part that I'm missing? Also, what does the two lines in the electrolyte mean? Is it separator?

Thanks a lot.

**Comment by valentinsulzer** [comment, marked as answer]
The three subdomains are negative electrode, separator, positive electrode. The vertical lines indicate the boundaries between these subdomains. The electrolyte permeates all three subdomains, it is not a separate domain. See section 2 of https://arxiv.org/pdf/2203.16091.pdf
